hoot-core/src/main/cpp/hoot/core/io/ApiDbReader.cpp

```diff
@@ -110,11 +171,8 @@ void ApiDbReader::_addTagsToElement(shared_ptr<Element> element)
         double tv = tags.getLength("accuracy").value();
         element->setCircularError(tv);
         ok = true;
-        LOG_TRACE(
-          "Set circular error from accuracy tag to " << tv << " for element with ID: " <<
-          element->getId());
       }
-      catch (const HootException& e)
+      catch (const HootException& /*e*/)
```

Comment:
Its to avoid an unused var warning.  Since nothing is being done with that exception, commenting out the var name will prevent the warning.